packages/react-instantsearch/src/core/InstantSearch.js

```diff
@@ -170,6 +177,8 @@ InstantSearch.propTypes = {
 
   createURL: PropTypes.func,
 
+  refresh: PropTypes.bool,
```

Comment:
You can put this one required, since `createInstantSearch` will always provide the prop.